src/main/webapp/WEB-INF/tags/instructor/results/filterEditModal.tag

```diff
@@ -61,6 +62,28 @@
                   </div>
                 </div>
               </c:if>
+              <div class="form-group">
+                <label id="sectionDisplayModeLabel" for="sectionDisplayMode" class="control-label">
+                  Include responses if:
+                </label>
+                <div data-toggle="tooltip" title="Specifies which responses should be marked as related to section">
+                  <select id="sectionDisplayMode" class="form-control" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_SECTION_DISPLAY_MODE%>">
+                    <option value="<%=SectionDisplayMode.BOTH_IN_SECTION%>"<c:if test="${filterPanel.sectionDisplayMode == 'BOTH_IN_SECTION'}"> selected</c:if>>
```

Comment:
Let's put the `<c:if>...</c:if>` on the next line. Applies to all subsequent `<option>` tags.